code/game/machinery/computer/camera_advanced.dm

```diff
@@ -182,10 +182,15 @@
 		CreateEye()
 
 	if(!eyeobj.eye_initialized)
-		var/camera_location
+		var/turf/camera_location
 		var/turf/myturf = get_turf(src)
-		if(eyeobj.use_static != FALSE)
-			if((!z_lock.len || (myturf.z in z_lock)) && GLOB.cameranet.checkTurfVis(myturf))
+		if(eyeobj.use_static)
```

Comment:
I think I reversed the condition. need to fix.
who the hell thought `value != FALSE` is a thing??